### PR TITLE
small additions

### PR DESCRIPTION
**Pull request by shedaniel**
enjoy now i can play bedwars

### CODE REVIEW

File: mappings/net/minecraft/client/gui/widgets/Button.mapping [hunk 1]
```diff
@@ -19,9 +19,20 @@ CLASS ke net/minecraft/client/gui/widgets/Button
...
 		ARG 3 y
 		ARG 4 text
+	METHOD a mouseReleased (II)V
+		ARG 1 mouseX
+		ARG 2 mouseY
 	METHOD a render (Lnet/minecraft/client/Minecraft;II)V
+		ARG 1 minecraft
+		ARG 2 mouseX
+		ARG 3 mouseY
+	METHOD a getYImage (Z)I
```
Comment (valoeghese) [inline mappings/net/minecraft/client/gui/widgets/Button.mapping]: I have no idea what the "y image" is meant to mean

Comment (shedaniel) [reply mappings/net/minecraft/client/gui/widgets/Button.mapping]: dw me neither

but it is from mojang names so this is good

File: mappings/net/minecraft/client/Minecraft.mapping [hunk 2]
```diff
@@ -16,5 +26,8 @@ CLASS net/minecraft/client/Minecraft net/minecraft/client/Minecraft
 	METHOD a start (Ljava/lang/String;Ljava/lang/String;Ljava/lang/String;)V
 		ARG 0 playerName
 	METHOD b getGameDirectory ()Ljava/io/File;
+	METHOD c printOpenGLError (Ljava/lang/String;)V
+	METHOD j toggleFullscreen ()V
+	METHOD l isLevelOnClient ()Z
```
Comment (liach) [inline mappings/net/minecraft/client/Minecraft.mapping]: hasLevel?

File: mappings/net/minecraft/container/DispenserContainer.mapping [hunk 3]
```diff
@@ -0,0 +1,2 @@
+CLASS sh net/minecraft/container/DispenserContainer
+	FIELD a entity Laz;
```
Comment (liach) [inline mappings/net/minecraft/container/DispenserContainer.mapping]: why is there an entity

File: mappings/net/minecraft/client/Minecraft.mapping (hunk 2 above)
Comment (shedaniel) [reply mappings/net/minecraft/client/Minecraft.mapping]: It also checks if the level is on the client which ugh yea I guess hasLevel is better

Comment (liach) [reply mappings/net/minecraft/client/Minecraft.mapping]:
```suggestion
	METHOD l hasLevel()Z
```

File: mappings/net/minecraft/container/DispenserContainer.mapping (hunk 3 above)
Comment (shedaniel) [reply mappings/net/minecraft/container/DispenserContainer.mapping]: because of good valo names that DispenserBlockEntity is DispenserEntity

Comment (liach) [reply mappings/net/minecraft/container/DispenserContainer.mapping]: nice, just apply that name suggestion above 
need this merged!

Comment (valoeghese) [reply mappings/net/minecraft/container/DispenserContainer.mapping]: `Good valo names`...
they're called cursed names excuzme

**Comment by valoeghese** [comment]
ready 2 merge?

**Pull request merged**